src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from "react";
+import LoginForm from "../../components/LoginForm/LoginForm";
+import logo from "../../media/logo190.png";
```

Comment:
I'd avoid "trash" folders. It's better to keep such things near to a component.

e.g.
```
pages/LoginPage/index.tsx
pages/LoginPage/logo.png
```